generate_variant_annotation.py

```diff
@@ -247,56 +122,72 @@ def main():
     )
 
     # Sort columns
-    col_order = ['locus_GRCh38', 'chrom_b37', 'pos_b37', 'chrom_b38', 'pos_b38',
-                 'ref', 'alt', 'allele_type', 'vep', 'rsid', 'af', 'cadd']
-    ht = ht.select(*col_order)
-
-    # Persist as writing twice would cause re-computation
-    ht = ht.persist()
```

Comment:
Since the table is outputted in more than one format, don't we want to persist the result before the first save?